Meetings/2022-02-16-meeting-minutes.md

```diff
@@ -16,7 +16,7 @@
 
 ## Recording
 
-**TBD**
+[2022-02-16 Meeting Recording](https://dotnetfoundation-my.sharepoint.com/:v:/g/personal/rob_prouse_dotnetfoundation_org/ERjxkEaTmpZLiRL7_fwXMKwB3aioi4OX26QwOG2eaLswyg?e=QMz63d)
```

Comment:
If you leave the board is this link still going to be active since it seems to be pointing to your personal account?

Reply:
I hope so. We previously tried sharing from a team account but that didn't work. I still have access to previous board member's files, but I'm not an O365 expert.

Reply:
This gets the files out there for now. I'll look for a better location though.

I just tried again from the DNF SharePoint without luck,

![image](https://user-images.githubusercontent.com/493828/157119662-fe9ce1ac-38d3-4cb8-a794-9b9de63af0d3.png)